bedrock/mozorg/templates/mozorg/about/governance/organizations.html

```diff
@@ -20,7 +20,7 @@ <h1 class="title-shadow-box">{{_('Mozilla Organizations')}} </h1>
 
 <p>{{_('The <a href="//www.mozilla.org/foundation/">Mozilla Foundation</a> is a California non-profit corporation exempt from Federal income taxation under IRC 501(c)(3). The Foundation supports the existing Mozilla community and oversees Mozilla’s governance structure. It also actively seeks out new ways for people around the world to recognize and steward the Internet as a critical public resource.')}}</p>
```

Comment:
While we're here, it would be good to fix the other protocol relative links. We can make all the mozilla.org links `https` explicitly. Even though they are not essentially "broken", using protocol relative links is now [considered an anti-pattern](http://www.paulirish.com/2010/the-protocol-relative-url/).